modules/ROOT/partials/configuration/browser_spellcheck.adoc

```diff
@@ -3,8 +3,6 @@
 
 This option configures {productname} to use the browser's native spell checker.
 
-`+browser_spellcheck+` is not available in either Internet Explorer 8 or 9, as neither browser has native spell checker functionality. For more complete browser support, we recommend using xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro].
```

Comment:
We probably don't want to lose the last sentence here? As we'd want to mention/highlight that Spell Checker Pro works better than native.

Reply:
The last sentence needs to be reworded a bit.

How about `For better spell checking functionality, we recommend using xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro]`

Reply:
For integrated spell-checking, we recommend xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro].

Reply:
`we` is a pronoun so isn't something we should use for technical documentation... see https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns. Maybe something like this instead:
```suggestion
For integrated spell-checking, checkout the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin.
```

Reply:
Another option would be to substitute `we` with `{companyname}` but I don't think that's the right solution here.